requirements/specifications/lifecycle/index.md

```diff
@@ -1,274 +1,365 @@
 # App Lifecycle Management
 
-Document Status: Proposed Specification
+Document Status: Working Draft
 
 See [Firebolt Requirements Governance](../../governance.md) for more info.
 
 | Contributor    | Organization   |
 | -------------- | -------------- |
 | Andrew Bennet            | Sky            |
+| Cody Bonney   | Charter |
+| Bart Catrysse            | Liberty           |
 | Tim Dibben            | Sky            |
+| Piotr Kobzda          | Liberty |
 | Jeremy LaCivita            | Comcast            |
+| Ramprasad Lakshminarayana | Sky |
 | Kevin Pearson            | Comcast            |
+| Jan Pedersen             | Sky |
 | Peter Yu            | Comcast           |
 
+
 ## 1. Overview
 
-This document describes the requirements that Firebolt platforms must
-fulfill when managing App Lifecycles. *App Lifecycle* refers to the
-lifecycle of an individual app from the time it is launched/loaded to
-the time it is unloaded and all runtime resources are discarded.
+This document describes the requirements that Firebolt platforms and
+Firebolt applications must fulfill when managing App Lifecycles. *App
+Lifecycle* refers to the lifecycle of an individual app from the time
+it is launched/loaded to the time it is unloaded and all runtime
+resources are discarded.
 
 *Loading* an app refers to fetching the initial resources, e.g. the
 app-manifest and index.html, and loading them into a container capable
 of running the app, e.g. a web browser.
 
-*Launching* an app refers to the process of getting an app into a state
-where it is some part of the user experience (e.g. visible, audible, or
-responding to input). This may include *loading* the app first, if
-needed.
+*Activating* an app refers to the process of getting an app into a state
+where it is presented as part of the user-perceptible experience (e.g.
+visible, audible, or responding to input). This may include *loading*
+the app first, if needed. For details on presentation, see [App
+Presentation](./presentation.md).
 
 *Closing* an app refers to the process of getting an app out of a state
 where it is the primary user experience (e.g not visible, not audible,
 and not responding to input). This **does not** involve *unloading* the
 app.
 
-*Unloading* an app refers to the process of notifying an app to do any
+*Suspending* an app refers to reducing the app's memory usage and CPU
+cycles, and deallocating its graphics compositors so that other apps
+will have more resources available.
+
+*Resuming* an app refers to reallocating its graphics compositors and
+resuming normal CPU and memory allocations.
+
+*Destroying* / unloading an app refers to the process of notifying an app to do any
 needed clean up, and then *terminating* it.
 
 *Terminating* an app refers to shutting down the app's container and
 reclaiming any memory, CPU, etc. that it was consuming.
 
+*Sleeping* an app refers to turning off all CPU cycles for that app
```

Comment:
"refers to putting the app into a state where it does not use CPU cycles, which allows the platform to optimise memory consumption.